src/lib/konnectors.js

```diff
@@ -146,15 +146,17 @@ export function run (cozy, konnector, account, timeout = 120 * 1000) {
       }
     }
   })
-  .then(job => waitForJobFinished(cozy, job, timeout))
+  .then(job => waitForJobFinished(cozy, job, timeoutToSuccess))
 }
 
 // monitor the status of the connector and resolve when the connector is ready
-function waitForJobFinished (cozy, job, timeout) {
+function waitForJobFinished (cozy, job, timeoutToSuccess) {
   return new Promise((resolve, reject) => {
     const idTimeout = setTimeout(() => {
-      reject(new Error('JOB_TIMEOUT'))
-    }, timeout)
+      clearInterval(idInterval)
```

Comment:
Good catch to clear `idInterval` here, but we should take care to define it before it is call, as a good practice. So I suggest to declare it as a variable juste before `idTimeout`:
```javascript
let idInterval
const idTimeout = setTimeout(() => {
// ...
```